docs/pages/components/TopBar.mdx

```diff
@@ -4,6 +4,7 @@ description: A container component that typically contains information about the
 source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/TopBar/TopBar.tsx
 ---
 
+import { useRef } from 'react'
```

Comment:
It looks like this can safely be removed.

Reply:
Removed here https://github.com/dequelabs/cauldron/pull/1680/commits/0b33e195335162011cbdafa79eb49c8d477b2435